doc/quick_start_guide.rst

```diff
@@ -173,13 +200,15 @@ For example:
 .. code-block:: bash
     python scripts/build_locally.py --target-hip=gfx90a
 
+Multi-target build
+~~~~~~~~~~~~~~~~~~
 
 It is, however, possible to build for Intel devices, CUDA devices, and an AMD device
```

Comment:
I believe it would be good to clarify, that the default build from the source enables support of Intel devices. But it is possible to extend that with support of both CUDA and AMD devices.